docs/03-node-operators/03-json-rpc/01-methods.md

```diff
@@ -1089,7 +1089,8 @@ Note that when `eth_estimateGas` is called, the node simulates the transaction e
 The simulation runs through the entire transaction process as if it were being executed, including checking for sufficient balance, contract code execution, etc.
 During the simulation, the method calculates the exact amount of gas that would be consumed by the transaction if it were to be executed on the blockchain. The estimated gas amount is returned, helping users set an appropriate gas limit for the actual transaction.
 
-There is a difference in Rootstock compared to Ethereum, and it is that if one of the steps of the simulated transaction fails, the node will return the gas estimation needed for the transaction, while on Ethereum, the node will return an error instead of the gas estimation.
+There is a difference in Rootstock compared to Ethereum, and it is that if one of the steps of the simulated transaction fails, the node will return the gas estimation needed for the transaction, while on Ethereum, the node will return an error instead of the gas estimation. Starting with Arrowhead 6.4.0 this is
+not valid anymore.
```

Comment:
question:

is it supposed to be `6.4.0` or should it be `6.5.0`? 

suggestion: 

Add a bit more info stating that from the new version, Rootstock will behave the same way as Ethereum. Something like:

>  Starting with Arrowhead 6.4.0 this isnot valid anymore. Rootstock will behave same way as Ethereum and return an error if one of the steps of the simulated transaction fails.

Reply:
Ended up rephrasing the paragraph a little bit adding what you suggested, I think it's really clear now:

> Until `Arrowhead 6.5.0`, there was a difference in Rootstock compared to Ethereum, and it is that if one of the steps of the simulated transaction fails, the node would return the gas estimation needed for the transaction, while on Ethereum, the node would return an error instead of the gas estimation. Starting with `Arrowhead 6.5.0` this is not valid anymore, Rootstock will behave same way as Ethereum, returning an error if one of the steps of the simulated transaction fails.

#TeamWork